code/modules/mob/living/carbon/human/species/synthetic.dm

```diff
@@ -16,8 +16,8 @@
 
 	total_health = 150 //more health than regular humans
 
-	brute_mod = 0.5
-	burn_mod = 0.9 //a small bit of resistance
+	brute_mod = 0.9
+	burn_mod = 0.9
```

Comment:
This doesn't need to change.

Reply:
It has to be done , synths are way too tanky for being a support role, check on any round and you will see synthetics abusing this to basically shrug off any xenomorph and do ballsy moves with no risk